lib/streamlit/config.py

```diff
@@ -394,6 +395,19 @@ def _global_unit_test():
 )
 
 
+@_create_option("server.cookieSecret")
+@util.memoize
+def _server_cookie_secret():
+    """Symmetric key used to produce signed cookies. If deploying on multiple
+    replicas, this must be set to ensure all replicas share the same secret.
```

Comment:
nitpick: change "must" to "should" because we don't detect multiple replicas and hence we don't report error if this is not set when running with multiple replicas

```suggestion
    replicas, this should be set to ensure all replicas share the same secret.
```